cli/flagstack/flagstack.go

```diff
@@ -0,0 +1,109 @@
+package flagstack
```

Comment:
nit: maybe call it "flag history"? "stack" initially made me think we'd be "popping" things off of it somehow :P

Reply:
sure sure!